reference/Upload-API.yaml

```diff
@@ -272,7 +272,7 @@ paths:
                 Not Found:
                   value:
                     status: 400
-                    message: Attribute id(s) e078d72a-aad4-4b68-9227-e0bde0b35xx1,e078d72a-aad4-4b68-9227-e0bde0b35xx2 not found
+                    message: 'Attribute id(s) e078d72a-aad4-4b68-9227-e0bde0b35xx1,e078d72a-aad4-4b68-9227-e0bde0b35xx2 not found'
```

Comment:
I swear this is Stoplight's magic, I did not touch any quotes 💢

Reply:
Yes not sure you need the quotes - doesn't seem to matter.